internal/api/api.go

```diff
@@ -216,6 +219,27 @@ func (api *idler) Info(w http.ResponseWriter, r *http.Request, ps httprouter.Par
 	}
 }
 
+func (api *idler) Reset(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+
+	logger := log.WithFields(log.Fields{"component": "api", "function": "Reset"})
+
+	openShiftAPI, openShiftBearerToken, err := api.getURLAndToken(r)
+	if err != nil {
+		logger.Error(err)
+		writeResponse(w, http.StatusBadRequest, fmt.Sprintf("{\"error\": \"%q\"}", err))
```

Comment:
there is already respondWithError that you can use.

Reply:
Yes, but I am planning to get rid of respondWithError. It is writeResponse + logging. Logging should be done outside of this function, as we will want print extra messages than error and we will also need to pass the logger if we want to add fields to it